var/spack/repos/builtin/packages/freefem/package.py

```diff
@@ -15,10 +15,11 @@ class Freefem(AutotoolsPackage):
     """
 
     homepage = "https://freefem.org"
-    url      = "https://github.com/FreeFem/FreeFem-sources/archive/refs/tags/v4.9.tar.gz"
+    url      = "https://github.com/FreeFem/FreeFem-sources/archive/refs/tags/v4.10.tar.gz"
```

Comment:
This isn't necessary (for future reference): the url is just a template.

Reply:
Okay

Reply:
It is possible to define different url? Just curious